### DISCUSSION TITLE
Problem with guzzle and php version using Laravel 8

### DISCUSSION THREAD

**Discussion by davworkx** [spatie/laravel-webhook-server#87] [Q&A]
Problem 1
    - spatie/laravel-webhook-server 1.4.0 requires guzzlehttp/guzzle ^6.3 -> found guzzlehttp/guzzle[6.3.0, ..., 6.5.x-dev] but it conflicts with your root composer.json require (^7.3).
    - spatie/laravel-webhook-server[1.5.0, ..., 1.10.0] require php ^7.4 -> your php version (7.3.4) does not satisfy that requirement.
    - spatie/laravel-webhook-server[1.11.0, ..., 1.13.0] require php ^8.0|^7.4 -> your php version (7.3.4) does not satisfy that requirement.
    - Root composer.json requires spatie/laravel-webhook-server ^1.4 -> satisfiable by spatie/laravel-webhook-server[1.4.0, ..., 1.13.0].

**Comment by jadShartouny** [comment]
same here, any fix for this?!
